frontend/public/scss/meta-product-page.scss

```diff
@@ -1,11 +1,5 @@
 @import "variables";
-@import "~@material/layout-grid/dist/mdc.layout-grid";
-@import "~@material/top-app-bar/dist/mdc.top-app-bar";
-@import "~react-picky/dist/picky";
-@import "~react-day-picker/lib/style";
-@import "~bootstrap/scss/bootstrap";
-@import "~video.js/dist/video-js";
-
+@import "~bootstrap/scss/bootstrap-utilities";
```

Comment:
Variables and the mixins available from bootstrap-utilities are the only pieces needed and neither contains css that will overwrite things when reimported because of the styles.js conditional import